pytorch_lightning/utilities/seed.py

```diff
@@ -51,7 +50,7 @@ def seed_everything(seed: Optional[int] = None) -> int:
         rank_zero_warn(f"{seed} is not in bounds, numpy accepts from {min_seed_value} to {max_seed_value}")
         seed = _select_seed_randomly(min_seed_value, max_seed_value)
 
-    log.info(f"Global seed set to {seed}")
+    rank_zero_info(f"Global seed set to {seed}")
```

Comment:
We chose to log on all ranks so users can be sure the same seed is used in all processes.

Reply:
Makes sense.